.changeset/config.json

```diff
@@ -8,7 +8,25 @@
   ],
   "commit": false,
   "ignore": ["livekit-agents-examples"],
-  "linked": [],
+  "fixed": [
+    [
+      "@livekit/agents",
+      "@livekit/agents-plugin-anam",
```

Comment:
nit: you could use a glob expr to match all plugins, if that's the intention. 
that way you don't have to remember to add it here when adding a new plugin